dotCMS/src/main/java/com/dotmarketing/business/PermissionBitFactoryImpl.java

```diff
@@ -133,49 +138,48 @@
 	 * 2. Permission type
 	 * 3. asset id
 	 */
-	private final String updatePermissionReferenceByAssetIdSQL = "update permission_reference set reference_id = ? where permission_type = ? and asset_id = ?";
+	private static final String UPDATE_PERMISSION_REFERENCE_BY_ASSETID_SQL = "update permission_reference set reference_id = ? where permission_type = ? and asset_id = ?";
 
 	/*
 	 * Select permission references based on how are referencing the type of reference
 	 * Parameters
 	 * 1. Reference id
 	 * 2. Permission type
 	 */
-	private final String selectPermissionReferenceSQL = "select asset_id from permission_reference where reference_id = ? and permission_type = ?";
+	private static final String SELECT_PERMISSION_REFERENCE_SQL = "select asset_id from permission_reference where reference_id = ? and permission_type = ?";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 154 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)